ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -843,36 +850,47 @@ let config_winbind_daemon ~workgroup ~netbios_name ~domain =
      * upgrade to samba packages with this capacity *)
     if !Xapi_globs.winbind_allow_kerberos_auth_fallback then "yes" else "no"
   in
+  let version_conf =
```

Comment:
I would prefer to see this inlined below such that the config file we generate can be seen as at a glance.